### DISCUSSION TITLE
Not accepting Google Authenticator code

### DISCUSSION THREAD

**Discussion by tonkaskippers** [pretix/pretix#2346] [Q&A] [answered]
**Describe the bug**
I setup two step authentication using Google Authenticator to secure my ticketing system.  It successfully set up.  However, it isn't accepting the codes anymore.   I put it on both admin accounts on my Pretix install.  Is there a way to disable it via SSH?

**Expected behavior**
I expect it to consistently accept the two step codes from Google Authenticator.
**Screenshots**
![image](https://user-images.githubusercontent.com/22083341/137828734-8dbf42ce-7c86-4e90-b6e9-1ea52028cc79.jpeg)

**Comment by raphaelm** [comment, marked as answer]
I'm not sure how you ended up in this situation but you can get out of it by going to the ``pretixbase_user`` table on your database and setting the ``require_2fa`` column to ``false``. Alternatively, you can create a new superuser with ``python -m pretix createsuperuser``.

**Comment by tonkaskippers** [comment]
Thanks!  That’s what I was looking for.

On Sun, Nov 28, 2021 at 10:37 AM Raphael Michel ***@***.***>
wrote:

> I'm not sure how you ended up in this situation but you can get out of it
> by going to the pretixbase_user table on your database and setting the
> require_2fa column to false. Alternatively, you can create a new
> superuser with python -m pretix createsuperuser.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/pretix/pretix/discussions/2346#discussioncomment-1712061>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AFIPODOI52EQKG25LPN2HKLUOJLEBANCNFSM5I5PFZBA>
> .
>
-- 
Sent from my iPhone
